.github/workflows/iroha2-label.yml

```diff
@@ -2,15 +2,20 @@ name: I2::Label
 
 on:
   pull_request_target:
-    branches: [iroha2-dev, iroha2-stable, iroha2-lts]
-    types: [opened]
 
 jobs:
-  add:
+  labeler:
+    permissions:
+      contents: read
+      pull-requests: write
     runs-on: ubuntu-latest
     steps:
-      - uses: actions-ecosystem/action-add-labels@v1
-        with:
-          github_token: ${{ secrets.github_token }}
-          labels: |
-            iroha2
+    - id: label-the-PR
+      uses: actions/labeler@v5
```

Comment:
https://github.com/actions/labeler

I think this is a good consolidation of `jitterbit/get-changed-files` and `actions-ecosystem/action-add-labels`